{
  "Header": {
    "Original": "CVE-2021-23177 (An improper link resolution flaw while extracting an archive can lead  ...)",
    "Line": 64300,
    "ID": "CVE-2021-23177",
    "Description": "(An improper link resolution flaw while extracting an archive can lead  ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-2987-1}",
      "Line": 64301,
      "Type": "xref",
      "Bugs": [
        "DLA-2987-1"
      ]
    },
    {
      "Original": "- libarchive 3.5.2-1 (bug #1001986)",
      "Line": 64302,
      "Type": "package",
      "Package": "libarchive",
      "Kind": "fixed",
      "Version": "3.5.2-1",
      "BugNo": 1001986
    },
    {
      "Original": "[bullseye] - libarchive 3.4.3-2+deb11u1",
      "Line": 64303,
      "Type": "package",
      "Release": "bullseye",
      "Package": "libarchive",
      "Kind": "fixed",
      "Version": "3.4.3-2+deb11u1"
    },
    {
      "Original": "[buster] - libarchive \u003cno-dsa\u003e (Minor issue)",
      "Line": 64304,
      "Type": "package",
      "Release": "buster",
      "Package": "libarchive",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/libarchive/libarchive/issues/1565",
      "Line": 64305,
      "Type": "NOTE",
      "Description": "https://github.com/libarchive/libarchive/issues/1565"
    },
    {
      "Original": "NOTE: https://github.com/libarchive/libarchive/commit/fba4f123cc456d2b2538f811bb831483bf336bad (v3.5.2)",
      "Line": 64306,
      "Type": "NOTE",
      "Description": "https://github.com/libarchive/libarchive/commit/fba4f123cc456d2b2538f811bb831483bf336bad (v3.5.2)"
    }
  ]
}